style.css

```diff
@@ -619,25 +579,197 @@ body {
   font-weight: 700;
   font-size: 40px;
   line-height: 52px;
-  margin-left: 50px;
-  margin-bottom: 20px;
 }
 
 .contact-text {
   font-style: normal;
-  font-weight: 400;
   font-size: 20px;
   line-height: 28px;
-  text-align: center;
   color: #ebebff;
-  margin-bottom: 40px;
-}
-
-.contact-form-background-image {
-  position: absolute;
-  width: 233px;
-  height: 670.17px;
-  left: 153px;
-  margin-left: -10px;
-  margin-top: 49px;
+  margin-right: 39px;
+  padding-right: 30px;
+  font-weight: 400;
+  width: 569px;
+  height: 102px;
+}
+
+/* ------------Desktop Version----------- */
+
+@media screen and (min-width: 768px) {
+  .body {
+    width: 90%;
+    background: #fff;
+  }
+
+  .toolbar {
+    display: flex;
+  }
+
+  .icons {
+    display: none;
+  }
+
+  .navu {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    width: 90%;
+  }
+
+  .navul1 {
+    margin-left: 150px;
+    margin-top: -20px;
+  }
+
+  .navul2 {
+    margin-top: -20px;
+    margin-left: 700px;
+    width: 50%;
+  }
+
+  .navulitems {
+    font-style: normal;
+    font-size: 15px;
+    line-height: 20px;
+    color: #344563;
+    display: flex;
+    flex-direction: row;
+    list-style-type: none;
+    padding-right: 100px;
+  }
+
+  .navli {
+    color: #344563;
+    margin: 5px;
+  }
+
+  .tool {
+    background-image: url(./Header\ bg.png);
+  }
+
+  .headline {
+    display: flex;
+    flex-direction: column;
+    width: 90%;
+    align-items: flex-start;
+    background-image: no-repeat;
+  }
+
+  .text1 {
+    flex-wrap: wrap;
+    width: 80%;
+    margin-left: 250px;
+  }
+
+  .text2 {
+    flex-wrap: wrap;
+    width: 60%;
+    margin-left: 250px;
+    margin-right: 20%;
+  }
+
+  .let {
+    margin-left: 250px;
+    margin-top: -60px;
+  }
+
+  .socialmediaicons {
+    width: 60%;
+  }
+
+  .leticon {
+    margin-top: -55px;
+    margin-left: 250px;
+  }
+
+  .card {
+    display: flex;
+    flex-direction: row;
+    margin-left: 15%;
+    padding-left: 50px;
+    width: 70%;
+  }
+
+  .card__image {
+    padding-left: 20px;
+    margin-left: 20px;
+    margin-right: 20px;
+    flex: 50%;
+    width: 100%;
+    height: 100%;
+  }
+
+  .card__content {
+    padding-right: 20px;
+    margin-left: 20px;
+    flex: 50%;
+    width: 100%;
+    height: 100%;
+    margin-right: auto;
+  }
+
+  .works {
+    background: #e5e5e5;
+    width: 100%;
+  }
+
+  .about {
+    margin-top: 100px;
+    padding-top: 10px;
+    display: flex;
+    flex-direction: row;
+    margin-left: 250px;
+    width: 80%;
+    padding-right: 100px;
+  }
+
+  .about__content {
+    align-content: flex-start;
+  }
+
+  .About__skills {
+    flex-direction: row;
+    margin-left: 100px;
+    color: #344563;
+  }
```

Comment:
Kindly check the position of the arrows, as well as the margins and spacing of the skills. The `framework` and `Skills` are too close to the languages part. Please check and follow the design dimensions in order to match it. :pray: 

Your design 

![Screenshot from 2022-12-15 21-55-16](https://user-images.githubusercontent.com/65233860/207979246-fb45f07c-a0fa-4723-962f-dfa7650f112b.png)


Figma design

![Screenshot from 2022-12-15 21-42-33](https://user-images.githubusercontent.com/65233860/207979329-0ec4f53b-3a69-4f7d-a27e-07f30320e899.png)